client/package.json

```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.2",
+    "chart.js": "^2.9.4",
```

Comment:
do we need to chart.js module, you install the react-chartjs-2 and use it at the Chart.js?